src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -652,6 +653,58 @@ describe('DetailsFieldsComponent', () => {
     componentToTest.onSubmit();
     expect(toastrServiceStub.error).not.toHaveBeenCalled();
   });
+
+  it('Should return a date in ISO format given a date, hour, minute, second and milisecond', () => {
```

Comment:
the test is too long, you could refactor this to reduce the size, you could define your constants outside and set the values in the beforeEach method or make a method that sets the values and then call it.

Reply:
I think it is unnecessary to define in the beforeEach because these values are only needed in this Test. You can see the AAA pattern to give you a clearer idea